app/property/model.py

```diff
@@ -0,0 +1,85 @@
+from app.db import db
+from app import json
+
+
+class Property(db.Model, json.Serialisable):
+    __tablename__ = 'property'
+
+    id = db.Column(db.Integer(), primary_key=True)
+    case_id = db.Column(db.Integer(),
+                        db.ForeignKey('case.id', ondelete="CASCADE"),
+                        nullable=False,
+                        unique=True)
+    title_number = db.Column(db.String(), nullable=False)
+    street = db.Column(db.String(), nullable=False)
+    extended = db.Column(db.String(), nullable=True)
+    locality = db.Column(db.String(), nullable=False)
+    postcode = db.Column(db.String(), nullable=False)
+    tenure = db.Column(db.String(), nullable=False)
+
+    def __init__(self,
+                 case_id,
+                 title_number,
+                 street,
+                 tenure,
+                 locality,
+                 postcode,
+                 extended=None):
+        self.case_id = case_id
+        self.title_number = title_number
+        self.street = street
+        self.extended = extended
+        self.locality = locality
+        self.postcode = postcode
+        self.tenure = tenure
+
+    def save(self):
+        db.session.add(self)
+        db.session.commit()
+
+    @staticmethod
+    def get(id_):
```

Comment:
Yes, sure. This pull request was created before we agreed to move them to service.